ast/compile.go

```diff
@@ -4049,6 +4049,11 @@ func safetyErrorSlice(unsafe unsafeVars, rewritten map[Var]Var) (result Errors)
 			v = w
 		}
 		if !v.IsGenerated() {
+			if _, ok := futureKeywords[string(v)]; ok {
+				result = append(result, NewError(UnsafeVarErr, pair.Loc,
+					"var %[1]v is unsafe (hint: `import future.keywords.%[1]v` to import a future keyword)", v))
```

Comment:
Could we also provide this hint during parsing?

```
> some x in [1,2,3]
|
2 errors occurred:
1:13: rego_parse_error: unexpected , token: expected rule key term (e.g., in[<VALUE>] { ... })
        some x in [1,2,3]
                    ^
1:13: rego_parse_error: unexpected , token: non-terminated rule key
        some x in [1,2,3]
                    ^
```

Reply:
Of course! I've missed that one...

Reply:
Not sure if this one can be deal with in the same, somewhat-generic way, though. But we should cover it nonetheless.